frontend/src/integrations/groupsio/config.js

```diff
@@ -0,0 +1,20 @@
+import config from '@/config';
+import GroupsioConnect from './components/groupsio-connect.vue';
+
+export default {
+  enabled: config.isGroupsioIntegrationEnabled,
+  hideAsIntegration: !config.isGroupsioIntegrationEnabled,
+  name: 'Groups.io',
+  backgroundColor: '#FFFFFF',
+  borderColor: '#FFFFFF',
+  description:
+    'Connect Groups.io to sync groups and topics activity.',
+  image:
+    '/images/integrations/groupsio.jpeg',
```

Comment:
Also, you will probably need to add an svg icon support to groups.io in `frontend/public/icons/crowd-icons.svg`. Otherwise, in the default reports, the groups.io icon will be blank

Reply:
Added svg